### PR TITLE
[terra-form-select]Fixed axe issue due to invalid Aria-activedescendant value.

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
#2765

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
Can we add a test that validates this issue is resolved?

**Comment by saket2403** [comment]
> Can we add a test that validates this issue is resolved?
Added Wdio test to validate an empty select field.

File: packages/terra-form-select/src/terra-dev-site/test/form-select/EmptySelectField.test.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import SelectField from 'terra-form-select/lib/SelectField';
+
+const SelectFieldExample = () => (
+  <SelectField label="T-shirt size" placeholder="Select a size" selectId="tshirt-size-field-2" required />
```
Comment (emilyrohrbough) [inline packages/terra-form-select/src/terra-dev-site/test/form-select/EmptySelectField.test.jsx]: I hate to be picky, but can we re-use an existing example like: https://engineering.cerner.com/terra-core/tests/terra-form-select/form-select/uncontrolled-search

we should be able to enter a bad value search result to generate "no results found"

Comment (saket2403) [reply packages/terra-form-select/src/terra-dev-site/test/form-select/EmptySelectField.test.jsx]: Reused test in ef3f17d

**Comment by PaulSolDev** [comment]
Functional review complete.  No issues identified.

**Pull request merged**